src/applications/letters/tests/actions/letters.unit.spec.js

```diff
@@ -36,20 +36,28 @@ import {
  * Teardown() resets it back to normal.
  */
 
+let oldWindow;
+
 const setup = () => {
   testkit.reset();
   mockFetch();
   setFetchJSONResponse(global.fetch.onCall(0), {});
+  oldWindow = global.window;
   global.window.URL = {
     createObjectURL: () => {},
     revokeObjectURL: () => {},
   };
 };
 
+const teardown = () => {
+  global.window = oldWindow;
```

Comment:
This doesn't _really_ restore `global.window` back to what is was before `.URL` was set on it. You need to do the full Object.create/Object.assign dance (ex: https://github.com/department-of-veterans-affairs/vets-website/blob/8e50552df8a4e5a5dadf613b8d36cc2fb2b78f1f/src/applications/post-911-gib-status/tests/actions/post-911-gib-status.unit.spec.js#L25-L32)

I noticed that we have quite a few cases where this isn't done correctly and it doesn't _seem_ to be causing trouble. 🤔

Reply:
Ah, ok. Thank you. If it's not causing trouble, we can close this PR and not do anything.

Reply:
sounds like a plan :)